messaging/messaging_batch.go

```diff
@@ -87,14 +88,138 @@ type BatchResponse struct {
 	Responses    []*SendResponse
 }
 
+// SendEach sends the messages in the given array via Firebase Cloud Messaging.
```

Comment:
Is this the comment that will be parsed into reference docs?

Normally, I would be asking for all literals to be backticked; but since that does not at all seem to be the standard in pages like https://pkg.go.dev/firebase.google.com/go@v3.13.0+incompatible/messaging#SendResponse I'll suggest the opposite, so our approach is more or less consistent.

So, I think things like SendAll()` and `BatchResponse` should actually *remove* the backticks.

In any case, we don't want a mix, like "An error
// from SendEach or a `BatchResponse` ...."

Any thoughts here Lahiru?  Thanks!

Reply:
Yep, these comments will appear in the reference docs. Go reference docs do not render backticks, but we have used them in some places in the past... I am good with us removing them here. Thanks, Eric!

Reference: https://pkg.go.dev/firebase.google.com/go/v4/messaging#Client.Send